lambdas/gro-convert-to-json/src/test/java/uk/gov/di/data/lep/ConvertToJsonTest.java

```diff
@@ -41,11 +43,32 @@ static void setup() throws JsonProcessingException {
         when(config.getGroRecordsBucketName()).thenReturn("JsonBucketName");
         when(awsService.getFromBucket(anyString(), anyString())).thenReturn(mockS3objectResponse);
 
-        event.detail = new S3ObjectCreatedNotificationEventDetail();
-        event.detail.bucket = new S3ObjectCreatedNotificationEventBucket();
-        event.detail.bucket.name = "XMLBucketName";
-        event.detail.object = new S3ObjectCreatedNotificationEventObject();
-        event.detail.object.key = "File.xml";
+        event = new S3ObjectCreatedNotificationEvent(
```

Comment:
Can this get inlined into the field definition/made final? It shouldn't be changing I think?

Reply:
sure